docs/resources/file.md.erb

```diff
@@ -51,10 +51,12 @@ The `content` property tests if contents in the file match the value specified i
 
 The following complete example tests the `pg_hba.conf` file in PostgreSQL for MD5 requirements.  The tests look at all `host` and `local` settings in that file, and then compare the MD5 checksums against the values in the test:
 
+    hba_config_file = '/etc/postgresql/9.1/main/pg_hba.conf'
```

Comment:
Can we just move the path into the describe block? I'm not a fan of showing this format and it would keep us uniform with the rest of this documentation.

Reply:
Sure. Happy to oblige.